tests/enterprise-contract/contract.go

```diff
@@ -388,31 +388,31 @@ var _ = framework.EnterpriseContractSuiteDescribe("Enterprise Contract E2E tests
 						ecp.SourceConfig{Include: []string{"attestation_task_bundle.task_ref_bundles_acceptable"}},
 					)
 					Expect(fwk.AsKubeAdmin.TektonController.CreateOrUpdatePolicyConfiguration(namespace, policy)).To(Succeed())
-		
+
 					generator.WithComponentImage("quay.io/redhat-appstudio/ec-golden-image:e2e-test-unacceptable-task")
 					pr, err := fwk.AsKubeAdmin.TektonController.RunPipeline(generator, namespace, pipelineRunTimeout)
 					Expect(err).NotTo(HaveOccurred())
 					Expect(fwk.AsKubeAdmin.TektonController.WatchPipelineRun(pr.Name, namespace, pipelineRunTimeout)).To(Succeed())
-		
+
 					pr, err = fwk.AsKubeAdmin.TektonController.GetPipelineRun(pr.Name, pr.Namespace)
 					Expect(err).NotTo(HaveOccurred())
-		
+
 					tr, err := fwk.AsKubeAdmin.TektonController.GetTaskRunStatus(fwk.AsKubeAdmin.CommonController.KubeRest(), pr, "verify-enterprise-contract")
 					Expect(err).NotTo(HaveOccurred())
-		
+
 					Expect(tr.Status.TaskRunStatusFields.Results).Should(Or(
 						// TODO: delete the first option after https://issues.redhat.com/browse/RHTAP-810 is completed
 						ContainElements(tekton.MatchTaskRunResultWithJSONPathValue(constants.OldTektonTaskTestOutputName, "{$.result}", `["FAILURE"]`)),
 						ContainElements(tekton.MatchTaskRunResultWithJSONPathValue(constants.TektonTaskTestOutputName, "{$.result}", `["FAILURE"]`)),
 					))
-		
+
 					//Get container step-report log details from pod
 					reportLog, err := utils.GetContainerLogs(fwk.AsKubeAdmin.CommonController.KubeInterface(), tr.Status.PodName, "step-report", namespace)
 					GinkgoWriter.Printf("*** Logs from pod '%s', container '%s':\n----- START -----%s----- END -----\n", tr.Status.PodName, "step-report", reportLog)
 					Expect(err).NotTo(HaveOccurred())
-					Expect(reportLog).Should(MatchRegexp(`Pipeline task .* uses an unacceptable task bundle`))
+					Expect(reportLog).Should(MatchRegexp(`Pipeline task .* uses an untrusted task bundle`))
```

Comment:
@zregvart , I think we can use `Or` here, like we do [here](https://github.com/redhat-appstudio/e2e-tests/blob/81d6508c88203bc5377c046700d67db459dd2968/tests/enterprise-contract/contract.go#L223-L227):

```go
Expect(reportLog).Should(Or(
    MatchRegexp(`Pipeline task .* uses an unacceptable task bundle`),
    MatchRegexp(`Pipeline task .* uses an untrusted task bundle`)
    ))
```

Either that or we can make a single regex that works with either.